src/sagemaker/mxnet/README.rst

```diff
@@ -76,6 +78,9 @@ For example, a training script might start with the following:
         parser.add_argument('--batch-size', type=int, default=100)
         parser.add_argument('--learning-rate', type=float, default=0.1)
 
+        # an alternative way to load hyperparameters via SM_HPS environmental variable.
+        parser.add_argument('--sm-hps', type=json.loads, default=os.environ['SM_HPS'])
```

Comment:
```
parser.add_argument('--sm-hps', type=dict, default=json.loads(os.environ['SM_HPS']))
```

Reply:
Question: I thought `type=json.loads` because it is an actual json string. Would `type=dict` handle situations like nested dictionaries or lists?

Reply:
Sorry, misread your comment. Let me do some background check again.

Reply:
@icywang86rui  I don't actually know this usage. What is the input format to work with `type=dict`?

I found that `type=dict` gave me errors, similar to this post: https://stackoverflow.com/questions/7625786/type-dict-in-argparse-add-argument

Reply:
when `type=dict` gave you errors, it was even with `json.loads(os.environ['SM_HPS'])` as the default? The Stack Overflow question seems to be about if you feed only a string to the parser when it's expecting a dict.